README.md

```diff
@@ -146,7 +146,7 @@ If you are using this library in one of your projects, add it in this list. :spa
 
  - [`gif-cli`](https://github.com/IonicaBizau/gif-cli)
 
- - [`ick`](https://github.com/nteract/ick#readme) by Kyle Kelley
+ - [`ick`](https://github.com/nteract/ick#readme) by [nteract devs](https://github.com/nteract)
```

Comment:
@rgbkrk Note these lines are not written by a human. They are written by a machine and will be overridden once I generate the README file again. :zap: 

To solve the root issue, change [this line](https://github.com/nteract/ick/blob/c1bb4f36d904495b500e52de7f774ba6371744d1/package.json#L12) into:

```
  "author": "nteract devs <...> (https://github.com/nteract)",
```